### DISCUSSION TITLE
How to use github copilot chat in company intenal network.

### DISCUSSION THREAD

**Discussion by WeBeautifulNow** [community/community#106883] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

we have installed copilot and copilot chat in our ide.
we need to use this ide in our company internal network env.
so we need to add some copilot endpoint into whitelist.

we have added https://copilot-proxy.githubusercontent.com/ in our whitelist. and we can use copilot now.
but for copilot chat, it still failed to connect.
is there any other url we need to add in whitelist for copilot chat?

**Comment by WeBeautifulNow** [comment]
Anybody could help?

**Comment by davecheney** [comment, marked as answer]
For chat, add `https://api.githubcopilot.com/chat/completions`
